lib/ansible/module_utils/gcp_utils.py

```diff
@@ -0,0 +1,120 @@
+# Copyright (c), Google Inc, 2017
+# All rights reserved.
+#
+# Redistribution and use in source and binary forms, with or without modification,
+# are permitted provided that the following conditions are met:
```

Comment:
Do you need the full license header in here for your needs?  We currently recommend using the following instead:

```
# Simplified BSD License (see licenses/simplified_bsd.txt or https://opensource.org/licenses/BSD-2-Clause)
```

Reply:
Sounds good. I've made the change.